src/tls/AEAD0.fst

```diff
@@ -7,8 +7,8 @@ module AEAD0
 //16-09-10 adapted from StreamAE and AEAD_GCM.
 //16-09-10 consider sharing some AEAD?.Common.fst
 
-open FStar.Heap
-open FStar.HyperHeap
+open Mem
+open Mem
 open FStar.Seq
```

Comment:
I just addressed your remark through new commit ecb849af297b808a787b927adb2ce25d77d694c7
on `taramana_mem`
It does not show up here -- probably because of trouble with VSTS status reporting.